ramlfications/loader.py

```diff
@@ -35,7 +42,88 @@ def _yaml_include(self, loader, node):
                 return inputfile.read()
 
         with open(file_name) as inputfile:
-            return yaml.load(inputfile, self._ordered_loader)
+            parsed = yaml.load(inputfile, self._ordered_loader)
+
+            if file_ext == ".json":
+                parsed = self._parse_json_refs(
+                    parsed,
+                    os.path.dirname(file_name),
+                )
+
+            return parsed
+
+    def _lookup_json_ref(self, ref_key, base_path=None, parent_schema=None):
+        """
+        Traverses the json pointer and returns the value of the pointer.
+        """
+        if "#" not in ref_key:
```

Comment:
Yeah. I think you're right. This should be fixed. Hmm. Maybe this is the right inflection point to differentiate internal vs. external refs?